lib/core/tree.js

```diff
@@ -121,6 +124,7 @@ class TernarySearchTree {
 
   /**
    * @param {Uint8Array} key
+   * @return {TstNode | null}
```

Comment:
```suggestion
   * @return {any}
```

TstNode does not return.